bin.src/convert_gen2_repo_to_gen3.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
+
+# This file is part of obs_base.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Convert a gen2 butler repo to gen 3. See
+`lsst.obs.base.ConvertRepoConfig` for most of the config options.
```

Comment:
Quick note. Please use the new scheme for writing scripts when you create new ones. We now have minimalist scripts in bin.src and place the argument parsing and main in the module itself. Please see https://community.lsst.org/t/new-tools-for-documenting-ad-hoc-scripts-in-the-lsst-science-pipelines/3784